File: Loan_Payment_Calculator.java
```java
import java.util.Scanner;

public class LoanPaymentCalculator {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter loan amount: ");
        double loanAmount = scanner.nextDouble();

        System.out.print("Enter loan term (in years): ");
        int termInYears = scanner.nextInt();

        System.out.print("Enter interest rate: ");
        double interestRate = scanner.nextDouble();

        double monthlyRate = interestRate / 100 / 12;
        double termInMonths = termInYears * 12;
        double monthlyPayment = (loanAmount*monthlyRate) / (1-Math.pow(1+monthlyRate, -termInMonths));

        System.out.printf("Monthly payment: %.2f\n", monthlyPayment);
    }
}
```

File: Loan_Payment_Calculator.py
def loan_payment_calculator():
    principal = float(input("Enter the loan amount: "))
    rate_of_interest = float(input("Enter the annual interest rate (in %): "))
    years = int(input("Enter the loan duration (in years): "))

    rate_of_interest = rate_of_interest / 100 / 12
    months = years * 12

    payment = (principal * rate_of_interest * (1 + rate_of_interest) ** months) / ((1 + rate_of_interest) ** months - 1)

    print("Your monthly payment will be: $%.2f" % payment)

loan_payment_calculator()